### DISCUSSION TITLE
Enabling Local instanceTicker?

### DISCUSSION THREAD

**Discussion by nihilsama** [misskey-dev/misskey#14872] [Q&A]
I wanted to enable seeing instance ticker for local users on my instance, I ended up on the store.ts file but I am confused on what value to change. 
![image](https://github.com/user-attachments/assets/f201513d-352c-4696-b8e7-5f46ec2c5a48)

**Comment by AmaseCocoa** [comment]
I think it will show up if you change “default: ‘remote’” to “default: ‘always’” (I have not verified this, but perhaps...)

**Reply by nihilsama** [reply]
this unfortunately didn't work, there are 3 store.ts files that contain this line, I changed all of them to no effect. thanks for the advice though
